discovery-provider/src/queries/queries.py

```diff
@@ -550,3 +551,11 @@ def get_creator_node_users():
         return api_helpers.success_response(users)
     except exceptions.ArgumentError as e:
         return api_helpers.error_response(str(e), 400)
+
+@bp.route("/l2/content_node", methods=("GET",))
+def get_l2_content_nodes():
```

Comment:
i would say here and throughout, we should probably not hard-code "l2" into the nomenclature here. this could exist on any chain, right? why do we want to call it l2?

Reply:
zero pref here, open to anything

Reply:
🤷  get_content_nodes?

Reply:
yeah my main concern was to avoid POA but also, given that primarily disco indexes L2 i can see just omitting that term and only using L1 in the minority case eg index_network_peers
nbd tho

Reply:
how would we feel about making it usrm_cnode or something to indicate it's as part of UserReplicaSetManager? bc L2 is also wrong if we migrate to a diff. chain like it might be 2 distinct L1 chains not eth +L2

Reply:
because that's really what this is - the current view of content nodes on UserReplicaSEtManager wherever that may live

Reply:
i think being specific about POA is not bad fwiw. like just having a poa specific endpoint and functions (as well as whatever chain we move to) will probably help us when thinking about backwards compat in the future